services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -161,6 +161,32 @@ export const SubmissionSummary = (): React.ReactElement => {
                 )}
 
                 <SubmissionTypeSummarySection
+                    subHeaderComponent={
+                        isCMSUser ? (
+                            <div className={styles.subHeader}>
+                                {pkg.mccrsID && (
+                                    <span>
+                                        MC-CRS record number:
+                                        <Link
+                                            href={`https://mc-crs.mathematica-stg.org/ReviewDetails/Index/${pkg.mccrsID}`}
```

Comment:
Looks like Meghan [commented](https://qmacbis.atlassian.net/browse/MCR-3547?focusedCommentId=124972) that the URL changed -  [https://mccrs.abtsites.com/Home/Index/[record](https://mccrs.abtsites.com/Home/Index/record) number]. Just commenting here so we don't forget